rocketc/rocketc.py

```diff
@@ -462,16 +463,31 @@ def create_group(self, data, suffix=""):
         LOG.info("Method Public Create Group: %s", group)
         return group
 
-    def _remove_user_from_group(self, group_name, user_id):
+    def _remove_user_from_group(self, group_name, user_id, auth_token=None):
         """
         This method removes a user form a team
         """
         api = self.api_rocket_chat
+
+        if group_name.startswith("Team-") and auth_token is not None:
+            regex = group_name.replace('Team-', '', 1)
+            query = {"name": {"$regex": regex}}
+            kwargs = {"query": json.dumps(query)}
+            api = self.api_rocket_chat
+            groups = api.list_all_groups(user_id, auth_token, **kwargs)
+
+            if groups["success"]:
+                groups = groups["groups"]
+                for group in groups:
+                    api.kick_user_from_group(user_id, group["_id"])
```

Comment:
No way to do this in just one request?

Reply:
The RocketChat api only has a method to remove a user from one group